lms/envs/common.py

```diff
@@ -340,9 +340,6 @@
     # Enable OpenBadge support. See the BADGR_* settings later in this file.
     'ENABLE_OPENBADGES': False,
 
-    # The block types to disable need to be specified in "x block disable config" in django admin.
-    'ENABLE_DISABLING_XBLOCK_TYPES': True,
```

Comment:
That's going to leave Open edX deployments in the lurch, though... Or require them to take extra actions before deploying this. It seems like having a data migration that simply replicates the current configuration would give a better migration/upgrade story.